resqpy/well/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""resqpy modules providing trajectory, deviation survey, blocked well, wellbore frame, wellbore marker frame and
+ md datum classes.
```

Comment:
Minor issue - first line needs to be a one-line, optionally follow by blank line and then any further lines.

A docstring like this that is almost a one-liner but is too long and run into two lines is unfortunately a common cause of ugly-looking HTML pages in the docs!

Reply:
![image](https://user-images.githubusercontent.com/71336336/143506011-175b3e8e-85e8-4eef-8f38-3d3e9dfebe8c.png)